woocommerce/woocommerce-gutenberg-products-block
# fix: accessibility of search products block

When output with php the block had correct accessibility on
the frontend but in the editor or when rendered as block
the correct aria label was missing because of misspelled
aria-label attribute.

Also fixed wrong value type provided for tabIndex in editor
while at it.




Fixes #1671




#### Accessibility



- [✔️  ] I've tested using only a keyboard (no mouse)
- [✔️ ] I've tested using a screen reader

### Testing

### Manual Testing

How to test the changes in this Pull Request:

In editor and on the frontend:

1. Verify using Axe extension ([Firefox](https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/), [Chrome](https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd)).
2. There should be no `Buttons must have discernible text` error in accessibility report

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).

With screenreader (CMD+F5 on OSX)

1. enable screen reader
2. focus on the search, tab to the button
3. the button should respond with `Search button` audio description



### Performance Impact



### Changelog

> ~Fixed accessibility label for submit Search button in Product Search block~ reverted in #5717
